layouts/html.hbs

```diff
@@ -126,6 +126,10 @@
       window.answersJWT = new Promise(resolve => {
         window.initAnswersJWT = resolve;
       });
+      const hhConfigMetadata = {
+        lastReceived: -1,
```

Comment:
What would you think about adding a `lastUpdatedTimestamp` or something like that to the `ConfigManager` itself? We could then use that in the comparison below.

Reply:
Sure, and include that data within the hitchhikerConfig data which is sent (as opposed to having it at the top level of the message)?